conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/ExplosionState.scala

```diff
@@ -21,16 +21,12 @@ import za.co.absa.enceladus.utils.explode.ExplosionContext
   * This class is used to encapsulate a state of exploded arrays during processing of dynamic conformance steps
   * by interpreters.
   *
-  * Interpreters such as MappingRuleInterpretedGroupExplode, ExplosionInterpreted and CollapseInterpreter
+  * Interpreters such as MappingRuleInterpreterGroupExplode, ExplosionInterpreter and CollapseInterpreter
   * can change this state.
   */
 class ExplosionState (ec: ExplosionContext = ExplosionContext()) {
 
-  private var _explodeContext: ExplosionContext = ec
-
-  def explodeContext: ExplosionContext = _explodeContext
-
-  def explodeContext_=(ec: ExplosionContext): Unit = _explodeContext = ec
+  var explodeContext: ExplosionContext = ec
```

Comment:
You can just have `class ExplosionState (var explodeContext: ExplosionContext = ExplosionContext())`
No need to reassign `ec` to `explodeContext`. 

Also the name `explodeContext` sounds like a funciton, because `explode` is a verb, why not call it `explosionContext` like its type?